GDJS/Runtime/events-tools/inputtools.js

```diff
@@ -127,14 +127,21 @@ gdjs.evtTools.input.keysNameToCode = {
   Numpad8: 104,
   Numpad9: 105,
 
-  RControl: 17,
-  RShift: 16,
-  RAlt: 18,
-  LControl: 17,
   LShift: 16,
+  RShift: 1016,
```

Comment:
With my proposal (see other comments), LShift becomes 1016, RShift 2016 (if I'm not mistaken).